resources/external_control.urscript

```diff
@@ -387,15 +462,15 @@ thread trajectoryThread():
         else:
           textmsg("Unknown spline type given:", raw_point[INDEX_POINT_TYPE])
           clear_remaining_trajectory_points()
-          socket_send_int(TRAJECTORY_RESULT_FAILURE, "trajectory_socket")
+          trajectory_result = TRAJECTORY_RESULT_FAILURE
         end
       end
     end
   end
   exit_critical
   stopj(STOPJ_ACCELERATION)
-  socket_send_int(TRAJECTORY_RESULT_SUCCESS, "trajectory_socket")
-  textmsg("Trajectory finished")
+  socket_send_int(trajectory_result, "trajectory_socket")
+  textmsg("Trajectory finished with result ", trajectory_result)
```

Comment:
Maybe convert the trajectory_result to a readable string before sending it to the log